Changes in module 'nf-core/fastqc'
--- modules/nf-core/fastqc/main.nf
+++ modules/nf-core/fastqc/main.nf
@@ -29,7 +29,7 @@
     printf "%s %s\\n" $rename_to | while read old_name new_name; do
         [ -f "\${new_name}" ] || ln -s \$old_name \$new_name
     done
-    fastqc $args --threads $task.cpus $renamed_files
+    fastqc -k $params.kmer $args --threads $task.cpus $renamed_files
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":

************************************************************
